src/components/Footer.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+
+import ExternalLink from "./ExternalLink";
+import NeighborIcon from "./NeighborIcon";
+
+const Footer = () => {
+    return (
+        <footer>
+            <small className="colophon">
+                Powered by{" "}
+                <ExternalLink to="https://www.notion.so">Notion</ExternalLink>
+                {", "}
+                <ExternalLink to="https://www.onfleet.com">
+                    OnFleet
+                </ExternalLink>{" "}
+                and an army of volunteers!
+            </small>
+
+            <ul>
```

Comment:
All SVGs are sticking out a little bit from their parent element, this should solve this 

```css
                ul li svg {
                    bottom: -3px; 
                    position: relative;
                }
```

Reply:
This isn't the svg — it's the anchor tag inside the list item. Is there something wrong with the way that it's spaced/looking/behaving currently?

Reply:
Because this is a little bit out of his parent it's not perfectly aligned